pkg/app/web/src/utils/is-nullish.ts

```diff
@@ -0,0 +1,7 @@
+export function isNullish(obj: unknown): boolean {
```

Comment:
looks like this function is not be used anywhere else 👀 should we remove it?
otherwise, I think `??` is good enough, no need to defined check `isNullish` ourselves. What do you think? 🤔 
ref https://mariusschulz.com/blog/nullish-coalescing-the-operator-in-typescript#providing-fallback-values-with-the-operator

Reply:
Ah, this file is not for this PR.
I'll remove this.
Thank you!